canonical-websites/www.ubuntu.com
# Bootstack page update

## Done
Updated the BootStack page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/cloud/managed-cloud](http://0.0.0.0:8001/cloud/managed-cloud)
- Check that the copy matches the copy doc

## Details
Copy doc: https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit
